lab_1_keywords_tfidf/main.py

```diff
@@ -4,20 +4,31 @@
 """
 from typing import Optional, Union
 
+from lab_1_keywords_tfidf.assets.freq_and_idf_dictionary_creation import file
+
+
+def clean_and_tokenize(text: str) -> [list[str]]:
+    name_file = 'Дюймовочка.txt'
+    File = open(name_file, 'rt')
+    text = File.read()
+    File.close()
```

Comment:
Work with files is already performed in `start.py` for you
Functions only process what was given as input arguments, in this case it is `text` variable

Reply:
Also, docstings must be placed _before_ the code
So it should be this way:
```
def clean_and_tokenize(text: str) -> Optional[list[str]]:
    Removes punctuation, casts to lowercase, splits into tokens
    Parameters:
    text (str): Original text
    Returns:
    Returns:  
    list[str]: A sequence of lowercase tokens with no punctuation

    In case of corrupt input arguments, None is returned
    <your code here>
```